app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -7,7 +7,7 @@ class Ovirt < ComputeResource
               :url_schema => ['http', 'https']
     validates :user, :password, :presence => true
     before_create :update_public_key
-    after_validation :update_available_operating_systems unless Rails.env.test?
+    after_validation :connect, :update_available_operating_systems unless Rails.env.test?
```

Comment:
If we do `#update_public_key` through `#connect` after each validation, do we still need to state it explicitly in `before_create`?

Reply:
you are right, not needed